src/scripts/main.js

```diff
@@ -2,7 +2,19 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const tElement = document.querySelector('tbody');
 
-// write your code here
+people.forEach(person => {
```

Comment:
```suggestion
people.forEach(({ sex, name, born, died, }) => {
```
better to use destructuring